telco-hub/configuration/reference-crs/optional/backup-recovery/README.md

```diff
@@ -0,0 +1,165 @@
+# Backup and Recovery process CR are documented here
+
+### Purpose 
+
+The purpose of this procedure is to show how to backup the Hub cluster CRs and  Recover in case of failure. It can either be in an active-passive Architecture or just a simple hub. 
+
+### Prerequisites 
+
+* OADP operator enabled through the MulticlusterHub CR ( RHACM)
+* An S3 Bucket ideally you want an s3 endpoint that is external to the hub. The purpose of this s3 bucket is to be used as a backup storage location
+for the multicluster-engine deployment of OADP. 
+
+### Backup Process
+
+#### 1-  Create OBC  and get the secret Information 
+
+```bash
+$ oc apply -f objectBucketClaim.yaml
+
+```
+```bash
+
+oc get obc
+```
+
+```yaml
+$ oc get secret -n default
```

Comment:
The OBC is created in the namespace `namespace: open-cluster-management-backup`. The secret should be there too, correct?